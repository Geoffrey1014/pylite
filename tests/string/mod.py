i = 123
s = "string"
print "i = %d" % i
print "s = %s" % s
print "i = %d, s = %s" % (i, s)
