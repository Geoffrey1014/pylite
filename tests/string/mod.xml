lambda
  name __init__
  param_count 0
  local_count 0
  label_count 0
  insn_vector
    load_const c0
    store_global c1
    load_const c2
    store_global c3
    load_global c1
    load_const c4
    mod
    print
    load_global c3
    load_const c5
    mod
    print
    load_global c3
    load_global c1
    new_tuple 2
    load_const c6
    mod
    print
    load_const c7
    return
  const_vector
    123.000000
    i
    "string"
    s
    "i = %d"
    "s = %s"
    "i = %d, s = %s"
    None
  guard_vector
  line_number_map 1 1 2 2 3 3 3 3 4 4 4 4 5 5 5 5 5 5 1 1
