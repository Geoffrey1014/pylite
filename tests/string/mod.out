i = 123
s = string
i = 123, s = string
